Userland/Libraries/LibArchive/Zip.cpp

```diff
@@ -68,11 +68,11 @@ Optional<Zip> Zip::try_create(ReadonlyBytes buffer)
         member_offset += central_directory_record.size();
     }
 
-    Zip zip;
-    zip.m_input_data = buffer;
-    zip.member_count = end_of_central_directory.total_records_count;
-    zip.members_start_offset = end_of_central_directory.central_directory_offset;
-    return zip;
+    return Zip {
+        end_of_central_directory.total_records_count,
```

Comment:
Any reason for not using designated initializes here as well? (btw the m_ prefix for input data is a bit weird, since this is not a C++ class)

Reply:
`Zip` is a class and the members are in the `private` section. Designated initializers can only be used to set public members.

I agree. The name is weird. I will change it.

Reply:
Changed.